src/views/forms/FormNodeEdit/FormNodeEdit.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="node-edit-form">
+    <ValidationObserver ref="observer" v-slot="{ handleSubmit }" slim>
+      <div>
+        <FormRow>
+          <template v-slot:label> {{ $t('todo_type') }}: </template>
+          <template v-slot:inputs>
+            <div class="select-container">
+              <Select v-model="formItems.todoType" class="select-size select-style">
+                <Option value="add-node">Connect to a custom node</Option>
+              </Select>
+            </div>
+          </template>
```

Comment:
What is the use case for this field? I see only an option to add a custom node

Maybe the label "Todo type" is misleading?

Reply:
> What is the use case for this field? I see only an option to add a custom node
> 
> Maybe the label "Todo type" is misleading?

Because this option will be expanded in the future